src/config/config.ts

```diff
@@ -8,9 +8,9 @@ export const limitForSavedFilters = 2;
 export const mail = {
   auth: {
     pass: process.env.MAIL_PASS,
-    user: process.env.MAIL_USER
+    user: process.env.MAIL_USER,
   },
-  service: 'Gmail'
+  service: 'Gmail',
```

Comment:
No-no :) we don't need a comma here

Reply:
Sorry, it's webstorm autoformater I forget about it =)